repo/packages/K/kubernetes/4/marathon.json

```diff
@@ -87,9 +107,8 @@
   "uris": {{kubernetes.uris}},
   "healthChecks": [
     {
-      "protocol": "HTTP",
-      "path": "/healthz",
-      "portIndex": 0,
+      "protocol": "COMMAND",
+      "command": { "value": "curl -f -k https://$HOST:{{kubernetes.apiserver-proxy-port}}/healthz" },
```

Comment:
I suspect that we could pass in a path to the CA cert and then not have to use the `-k` flag